params/networks.go

```diff
@@ -0,0 +1,51 @@
+package params
+
+import "errors"
+
+// GetNetwork returns the network of the current build.
+func GetNetwork() Network {
+	return network
+}
+
+// DefaultNetwork sets a default network for Celestia Node.
+var DefaultNetwork = DevNet
+
+// NOTE: Every time we add a new long-running network, it has to be added here.
+const (
+	// DevNet or devnet-2 according to celestiaorg/networks
+	DevNet Network = "devnet-2"
```

Comment:
IMO, this should rather be in a config file instead of code. Which also means it would be set at run-time. For bridge-nodes the config could be shared between core/app and node (chain-id).

Reply:
Initialized node can't be easily switched to another network over the same root folder/Store. If we give this opportunity in the config that means we should support it and test it, otherwise users will do this and break/corrupt their Store and Node. Another safe option we discussed with Rene is to pass the network name on init and initialize with it.